addons/uom/static/src/components/many2x_uom_tags/many2x_uom_tags.js

```diff
@@ -13,6 +13,15 @@ import { UomAutoComplete } from "@uom/components/uom_autocomplete/uom_autocomple
 import { roundPrecision } from "@web/core/utils/numbers";
 import { onWillUpdateProps } from "@odoo/owl";
 
+function _getProductRelatedModel() {
+    const field = this.env.model.config.fields[this.props.productField]
```

Comment:
For some reason, this returns the fields of the parent model not the model containing the `many2one` field (for example, `sale.order` instead of `sale.order.line`), which leads to an error in almost all the forms that use this widget. I'm not sure if there is a way to overcome this and retrieve info about the fields of the correct model.